ci/requirements/environment-windows.yml

```diff
@@ -1,20 +1,16 @@
 name: xarray-tests
 channels:
   - conda-forge
-  - nodefaults
 dependencies:
-  - python=3.6
-  - black
   - boto3
   - bottleneck
   - cartopy
-  - cdms2
-  - cfgrib
+  # - cdms2  # Not available on Windows
+  # - cfgrib  # Causes Python interpreter crash on Windows
```

Comment:
is there some issue we can link to?

Reply:
This change appears to have been added in https://github.com/pydata/xarray/pull/3340 (a year ago). I am going to enable `cfgrib` on windows for the time being to see if the issue still persists today.

Reply:
The issue still exists: https://github.com/pydata/xarray/runs/1653831281?check_suite_focus=true